src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -450,6 +510,17 @@ protected void outputException(X x, HttpServletResponse resp, Format format, Pri
       outputterHtml.outputEndTable();
       outputterHtml.outputEndHtml();
       out.println(outputterHtml.toString());
+    }else if ( format == Format.JSONJ ) {
```

Comment:
space between } and 'else'

Reply:
hard to tell from this PR, what is the difference between this JSONJ block and that above?

Reply:
This JSONJ block is for the outputException method which is called when ever there is an error. Without this block the webpage won't load and you would have no idea if it failed or not. The JSONJ block above is for the put command.